docs/04-guides/add-viewership/01-dashboard.mdx

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: dashboard
+title: Dashboard
+pagination_label: "Add viewership"
```

Comment:
This guide is called "Add viewership" but the dashboard section just shows you were to find the total views of a playback. Let's reconsider this guides title and scope. Perhaps "Get unique viewer counts for a video asset"?

Reply:
Yeah - this is a section that I'm not too sure about. Do we want to have a dashboard section? IMO it seems pretty unnecessary

Reply:
Yeah right now it seems unnecessary but over time I think it will make more sense as we eventually support more engagement metrics with detailed charts. It will live inside an analytics tab to the left of the health tab. YouTube Studio, for example, has a doc that goes over its analytics dashboard. https://support.google.com/youtube/answer/9002587?hl=en

Reply:
Yeah with that context it makes sense to have it then